src/apiCalls.js

```diff
@@ -1,3 +1,3 @@
-export default function fetchData() {
-    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies').then(response => response.json()).catch(error => console.log(error))
+export default function fetchData(path) {
```

Comment:
I love to see a flexible, reusable fetch! I do think this function name could be a bit more descriptive about what kind of data we're getting, though!